packages/@react-aria/form/src/useFormValidation.ts

```diff
@@ -75,25 +115,64 @@ export function useFormValidation<T>(props: FormValidationProps<T>, state: FormV
     state.commitValidation();
   });
 
+  let onBlur = useEffectEvent((event: Event) => {
+    const input = ref?.current;
+    const relatedTarget = (event as FocusEvent).relatedTarget as Element | null;
+    if (
+      (!input || !input.validationMessage) ||
+      (relatedTarget && isValidatableElement(relatedTarget) && (relatedTarget as ValidatableElement).validationMessage)
+    ) {
+      // If the input has no validation message,
+      // or the relatedTarget has a validation message, don't announce the error message.
+      // This prevents announcing the error message when the user is navigating
+      // between inputs that may already have an error message.
+      return;
+    }
```

Comment:
just to make sure I'm understanding the expected behavior here: 
the polite error message will announce if you navigate to a field that isn't in an error state from one that was invalid but NOT if you are going between multiple invalid fields since we prefer the user to fix the error on the field they just landed on?